quaint/src/ast/compare.rs

```diff
@@ -37,19 +37,14 @@ pub enum Compare<'a> {
     /// without visitor transformation in between.
     Raw(Box<Expression<'a>>, Cow<'a, str>, Box<Expression<'a>>),
     /// All json related comparators
-    #[cfg(any(feature = "postgresql", feature = "mysql"))]
```

Comment:
Discussed in https://prisma-company.slack.com/archives/C058VM009HT/p1709134435703369